css/modules.css

```diff
@@ -59,30 +60,80 @@ body .header {
   clear: both;
 }
 
-/*Creating Slideshow and setting locations
-Later, this will be put into @media for larger screens
-=============================================*/
-.slideshow-div{
-  margin: 0px 5%;
-  width: 100%;
-  overflow: hidden;
-}
-#slideshow-article .slideshow{
-  /*display: block;*/
-  width: 90%;
-  margin: 10px 0px;
-  padding: 0px;
-}
-#slideshow-article .my-slides{
-  display: inline;
-  float: left;
-  list-style-type: none;
-  /*max-width: 100%;
-  min-height: 150px;
-  max-height: 300px;*/
+/*Desktop/non-ghetto tablets Compatibility
+=========================================================================================*/
+@media screen and (max-width: 740px) {
+  /*Adds an rgba background with opacity to every div element on the page
+  =============================================*/
+  .divs {
+    border: solid black;
+    margin: 30px 0px;
+    background-color: rgba(200, 200, 200, 0.70);
+  }
+  /*header and nav-bar special borders, fixed nav bar
+  =============================================*/
+  .nav-bar {
+    display: none;
+  }
+  .nav-burger{
+    font-size: 1em;
+    color: rgba(013,013,013, .85);
+    float: right;
+  }
+  body .header {
+    display: block;
+    position: absolute;
+    min-width: 85vw;
+    height: 5vh;
+    margin-top: 5.2vh;
+    padding-left: 0;
+    padding-right: 0;
+    text-align: left;
+    font-size: 1.7em;
+    border: 2px dashed #FF0000;
+    background-color: rgba(200, 200, 200, 0.70);
+  }
+  /*table of contents for the projects floated to the left of the slideshow
+  =============================================*/
+  .table-of-contents{
+    display: block;
+    text-align: center;
+    margin-top: 0;
+    min-width: 17.5vw;
+    max-width: 17.5vw;
+    font-size: 0.75em;
+    /*overflow: hidden;*/
+  }
+  .table-of-contents li{
+    margin-left: 1.15em;
+    text-align: left;
+    list-style: lower-roman;
+    font-size: 0.6em;
+  }
+  .main-intro{
+    float: left;
+    margin-top: 0;
+    margin-left: .5vw;
+    margin-right: 0px;
+    min-width: 65vw;
+    max-width: 65vw;
+  }
+  #slideshow-article {
+    display: block;
+    width: 85vw;
+    margin-top: 12.5vh;
+  }
+  #slideshow-article img{
+    background: rgba(013,013,013, .85);
+  }
+  /* Clear footer and projects from floats slideshow
+  =============================================*/
+  .projects footer{
+    clear: both;
+  }
 }
 
-/*Desktop Compatibility
+/*Desktop/non-ghetto tablets Compatibility
 =========================================================================================*/
 @media screen and (min-width: 740px) {
```

Comment:
Make sure the parts of this you want to apply both to smaller and larger screens is not inside a @media query. I would put all styling that should stay the same throughout at the top before the @media queries.